src/packages/client/fixtures/blog/prisma/schema.prisma

```diff
@@ -4,8 +4,9 @@ datasource db {
 }
 
 generator client {
-  provider      = "prisma-client-js"
-  binaryTargets = ["native"]
+  provider             = "prisma-client-js"
+  binaryTargets        = ["native"]
+  experimentalFeatures = "transactionApi"
```

Comment:
this should be an array probably

Reply:
Yes, this is just a string in the meantime until the datamodel parser supports arrays here.